string uuid
string data
time start_time
time end_time
